client/blocks/blog-stickers/list.jsx

```diff
@@ -0,0 +1,22 @@
+/**
+ * External Dependencies
+ */
+import React from 'react';
+import { map } from 'lodash';
+import { localize } from 'i18n-calypso';
+
+const BlogStickersList = ( { stickers } ) => {
+	return (
+		<div className="blog-stickers__list">
+			<h3 className="blog-stickers__list-title">Blog stickers</h3>
+			<ul className="blog-stickers__list-ul">
+				{ map( stickers, sticker => {
+					const key = `blog-sticker-${ sticker }`;
```

Comment:
this can just be sticker. the key only has to be unique for this particular list.